src/main.js

```diff
@@ -461,6 +465,19 @@ function handleAppWebContentsCreated(dc, contents) {
 
   // implemented to temporarily help solve for https://community-daily.mattermost.com/core/pl/b95bi44r4bbnueqzjjxsi46qiw
   contents.on('before-input-event', (event, input) => {
+    if (input.key === 'Alt' && input.type === 'keyUp' && altLastPressed) {
+      altLastPressed = false;
+      mainWindow.webContents.send('focus-three-dot-menu');
+      return;
+    }
+
+    // Hack to detect keyPress so that alt+<key> combinations don't default back to the 3-dot menu
+    if (input.key === 'Alt' && input.type === 'keyDown') {
+      altLastPressed = true;
+    } else {
+      altLastPressed = false;
+    }
```

Comment:
why is ALT a shorcut? wouldn't that interfere not only with alt+up/down but with anything regarding the OS from the user if he has any shortcut using that? whouldn't it make more sense to use a separate shortcut for that that doesn't use a modifier key as the only one?
non-comprehensive list of things that might break:
- alt+f4 on windows (as an example of OS related)
- alt+right click for mark as unread (as an example of MM related)

Reply:
It's a manual override to do something the OS already does, which is open a window context menu  when you press ALT. It was part of @esethna's suggestions for accessibility.

I agree it would be better to use a different shortcut, but that's why we're checking for keyup and keydown to make sure it's not being used as a combo.